pyiron_atomistics/atomistics/structure/neighbors.py

```diff
@@ -524,6 +526,58 @@ def _check_width(self, width, pbc):
                 return True
         return False
 
+    def get_spherical_harmonics(self, l, m, cutoff_radius=np.inf, rotation=None):
+        """
+        Args:
+            l (int/numpy.array): Degree of the harmonic (int); must have ``l >= 0``.
+            m (int/numpy.array): Order of the harmonic (int); must have ``|m| <= l``.
+            rotation (numpy.array/list): Rotation to make sure phi does not become nan
```

Comment:
Should document shape of `rotation`.  It makes sense as a rotation matrix, but it should be mentioned, otherwise I see myself already checking source in a few months. ;P

Reply:
What's the common notation? Do I say `3x3 numpy.array/list` or should it come after the colon?

Reply:
Numpy does it this way
```
a : (..., M, M) array_like
    Input array to compute determinants for.
```
(from numpy.linalg.det). I'd just follow like this
```suggestion
            rotation ( (3,3) numpy.array/list): Rotation to make sure phi does not become nan
```